Resources/input/fsh/ISiKPatient.fsh

```diff
@@ -16,104 +32,313 @@ Description: "Dieses Profil beschreibt die Nutzung von administrativen Patienten
+  * ^short = "Lebenslange Krankenversichertennummer"
+  * ^comment = "Die als 'KVZ10' kodierte Versichertennummer ist der 10-stellige, 
+    unveränderbare Teil der Versichertennummer, 
+    der lesbar auf die Elektronische Gesundheitskarte aufgedruckt ist.
+    Er gilt für alle Krankenversichertennummern, 
+    unabhängig davon, ob es sich um GKV, PKV oder Sonderkostenträger handelt.  
+    **Weitere Hinweise:** siehe [Deutschen Basisprofile](https://simplifier.net/guide/leitfaden-de-basis-r4/ig-markdown-LebenslangeKrankenversichertennummer10-stelligeKVID-Identifier?version=current)"
   * type 1.. MS
+    * ^short = "Art des Identifiers"
+    * ^comment = "Hier ist stets der Code `KVZ10` 
+    aus dem CodeSystem `http://fhir.de/CodeSystem/identifier-type-de-basis` anzugeben.  
+    **Begründung Pflichtfeld:** `type` dient der Unterscheidung verschiedener Identifier-Typen"  
   * system MS
-  * value MS    
+    * ^short = "Namensraum der Versichertennummer"
+    * ^comment = "Hier ist stets der Wert `http://fhir.de/sid/gkv/kvid-10` anzugeben.  
+    **Begründung Pflichtfeld:** `system` stellt in Kombination mit `value` die Eindeutigkeit eines Identifiers sicher."
+  * value MS 
+    * ^short = "Lebenslange Krankenversichertennummer"
+    * ^comment = "Der 10-stellige, unveränderbare Teil der Versichertennummer."  
 * identifier[VersichertenId-GKV] only IdentifierKvid10
   * ^patternIdentifier.type = $identifier-type-de-basis#GKV
-  * ^comment = "Die Verwendung der 'GKV'-Kodierung einer Versichertennummer ist abgekündigt. Bitte den 'VersichertenId'-Slice verwenden."
+  * ^short = "Gesetzliche Krankenversichertennummer"
+  * ^comment = "**WARNUNG**: Die Verwendung der 'GKV'-Kodierung einer Versichertennummer ist abgekündigt,
+  da die lebenslangen Versichertennummer ab 2024 auch für PKV oder Sonderkostenträger eingeführt wird. 
+  Bitte statt dessen künftig den 'VersichertenId'-Slice verwenden."
   * type 1.. MS
+    * ^short = "Art des Identifiers"
+    * ^comment = "Hier ist stets der Code `GKV` 
+    aus dem CodeSystem `http://fhir.de/CodeSystem/identifier-type-de-basis` anzugeben.  
+    **Begründung Pflichtfeld:** `type` dient der Unterscheidung verschiedener Identifier-Typen"
   * system MS
+    * ^short = "Namensraum der Versichertennummer"
+    * ^comment = "Hier ist stets der Wert `http://fhir.de/sid/gkv/kvid-10` anzugeben.  
+    **Begründung Pflichtfeld:** `system` stellt in Kombination mit `value` die Eindeutigkeit eines Identifiers sicher."
   * value MS
+    * ^comment = "Enthält den eigentlichen Wert des Identifiers.  
+    **Begründung Pflichtfeld:** Ist der Wert nicht bekannt, sollte der gesamte Slice weggelassen werden."
 * identifier[Patientennummer] only IdentifierPid
   * ^patternIdentifier.type = $v2-0203#MR
+  * ^short = "Organisationsinterner Patienten-Identifier (PID)"
+  * ^comment = "Organisationsintere Patienten-Identifier werden von z.B. von KIS-Systemen vergeben 
```

Comment:
Issue: Bitte um Prüfung. Wie sieht es im Falle aus, dass ein Client einen Patienten anlegt? Dieser Fall ist im Terminplanungs-Modul denkbar. Wie soll das in der Praxis funktionieren? Ein Client kann die interne ID ja nicht kennen. @simoneOnFhir

Reply:
Ist korrekt, streng genommen bräuchte das Terminplanungsmodul dann ein eigenes Patienten-Profil *oder* wir machen den Identifier optional, stellen aber in den Kommentaren klar, dass dieser  von einem Server IMMER geliefert werden muss und das in den Testszenarien auch entsprechend gepfrüft wird.

Leider können wir soetwas nicht über Invarianten lösen, da es kein Flag in der Ressource gibt, an das wir eine "wenn du Client/Server bist, dann..."-Bedinung hängen könnten